src/providers/caching-gas-provider.ts

```diff
@@ -41,8 +44,8 @@ export class CachingGasStationProvider extends IGasPriceProvider {
     }
 
     log.info('Gas station price local cache miss.');
```

Comment:
i think we should delete this too btw, we dont need an info here

Reply:
Agree.. Kinda silly imho.. If we want gas price cache hit/miss ratio, we should log metrics.